canvas_modules/common-canvas/__tests__/toolbar/toolbar-test.js

```diff
@@ -107,15 +107,90 @@ describe("Toolbar renders correctly", () => {
 			.simulate("click");
 		expect(toolbarActionHandler.calledOnce).to.equal(false);
 	});
+
+	it("should render a Toolbar with medium size buttons", () => {
+		const toolbarConfig = {
+			leftBar: [
+				{ action: "palette", label: "Palette", enable: true },
+				{ divider: true },
+				{ action: "stop", label: "Stop Execution", enable: false },
+				{ action: "run", label: "Run Pipeline", enable: false },
+			]
+		};
+		const canvasToolbar = createToolbar(toolbarConfig, sinon.spy(), "md");
+		// Select the toolbar medium buttons
+		const overflowButtons = canvasToolbar.find(".toolbar-overflow-item button");
+		const defaultButtons = canvasToolbar.find(".toolbar-item.default button");
+
+		expect(overflowButtons).to.have.length(3);
+		expect(defaultButtons).to.have.length(3);
+
+		// Verify if the buttons show up with medium size
+		expect(overflowButtons.find(".bx--btn--md")).to.have.length(3);
+		expect(defaultButtons.find(".bx--btn--md")).to.have.length(3);
+	});
+
+	it("should render a Toolbar with small size buttons", () => {
+		const toolbarConfig = {
+			leftBar: [
+				{ action: "palette", label: "Palette", enable: true },
+				{ divider: true },
+				{ action: "stop", label: "Stop Execution", enable: false },
+				{ action: "run", label: "Run Pipeline", enable: false },
+			]
+		};
+		const canvasToolbar = createToolbar(toolbarConfig, sinon.spy(), "sm");
+		// Select the toolbar small buttons
+		const overflowButtons = canvasToolbar.find(".toolbar-overflow-item button");
+		const defaultButtons = canvasToolbar.find(".toolbar-item.default button");
+
+		expect(overflowButtons).to.have.length(3);
+		expect(defaultButtons).to.have.length(3);
+
+		// Verify if the buttons show up with small size
+		expect(overflowButtons.find(".bx--btn--sm")).to.have.length(3);
+		expect(defaultButtons.find(".bx--btn--sm")).to.have.length(3);
+	});
+
+	it("should render a Toolbar buttons with only icons ", () => {
+		const toolbarConfig = {
+			leftBar: [
+				{ action: "cut", enable: true, incLabelWithIcon: "none" },
+			]
+		};
+		const canvasToolbar = createToolbar(toolbarConfig);
+		// Select the toolbar only icons buttons
+		const defaultButtons = canvasToolbar.find(".toolbar-item.default button");
+
+		expect(defaultButtons).to.have.length(1);
+
+		// Verify if the buttons show up with icon-only carbon class
+		expect(defaultButtons.find(".bx--btn--icon-only")).to.have.length(1);
+	});
+
+	it("should render a Toolbar buttons with icon&label ", () => {
+		const toolbarConfig = {
+			leftBar: [
+				{ action: "run", label: "Label btn", enable: true },
+			]
+		};
+		const canvasToolbar = createToolbar(toolbarConfig);
+		// Select the toolbar only icons buttons
+		const defaultButtons = canvasToolbar.find(".toolbar-item.default button");
+
+		expect(defaultButtons).to.have.length(1);
+		expect(defaultButtons.find(".bx--btn--icon-only")).to.have.length(1);
```

Comment:
This test says testing icon with label yet it is expecting to find `".bx--btn--icon-only"`